css/index.css

```diff
@@ -143,4 +143,56 @@ footer nav {
 footer nav a {
     color: white;
     text-decoration: none;
+}
+
+/* header and image styles*/
+header, .imgContainer {
+    width: 800px;
+    margin: 2% auto;
+}
+header {
+    display: flex;
+}
+.logo {
+    width: 20%;
+}
+nav {
+    width: 80%;
+    display: flex;
+    justify-content: space-around;
+}
+nav a{
+    color: #4f4f4f;
+    text-decoration: none;
+    padding-top: 3%;
+}
+.middle-content .boxes .box:nth-child(1)  {
```

Comment:
Great job! Another method would be to give all of the boxes the same class and each box a different id. This will allow you to style all of the boxes collectively while giving each box their own color.